### PR TITLE
Add more experiments

### PR DESCRIPTION
**Pull request by echarles**
Related to https://github.com/jupyterlab/benchmarks/issues/49

This PR adds some experiments to find the root cause of the performance bottlenecks when displaying a lot of editors.

It implements similar display logic for various editors and measures:

```
Comparison for `DOMContentLoaded` (seconds) for 1000 editors.

- 1000 JupyterLab Cells: 50 seconds.
- 1000 JupyterLab Cells in a Lumino BoxPanel: 10 seconds for DOMContentLoaded, cells are only shown after 140 seconds.
- 1000 CodeMirrors: 50 seconds.
- 1000 Monaco: 40 seconds for DOMContentLoaded, editors are only shown after 60 seconds.
- 1000 ProseMirror (basic version): 2 seconds.
- 1000 ReMirror: 50 seconds.
```

### CODE REVIEW

File: experiments/cell/notebooks/hello.ipynb [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{
```
Comment (goanpeca) [inline experiments/cell/notebooks/hello.ipynb:1]: Maybe a more descriptive name for the notebook could help?

File: experiments/cell/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      initialFactories: standardRendererFactories
+    }),
+    model: new CodeCellModel({})
+  }).initializeState();
+  cells.push(cell);
+}
+
+cells.forEach((cell, index) => {
+  cell.activate();
+  console.log(`activate ${index}`, performance.now());
```
Comment (goanpeca) [inline experiments/cell/src/index.ts:26]: where does the `performance.now()` come from 🙃 ?

File: experiments/cell/style/index.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+body {
+  background: white;
+  margin: 0;
+  padding: 0;
+}
+
+.jp-CodeCell {
+  min-height: 50px;
+/*  content-visibility: auto; */
```
Comment (goanpeca) [inline experiments/cell/style/index.css:14]: Are we finally using this or moving on without it?

File: experiments/monaco/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# JupyterLab Benchmarks
+
+A page with may prosemirror editors.
```
Comment (goanpeca) [inline experiments/monaco/README.md:3]: Should this be monaco editors instead of prosemirror?

**Review by goanpeca — COMMENTED** [review]
Left a couple of minor comments and a couple of questions. Otherwise looks good. Great work @echarles!

File: experiments/cell/src/index.ts (hunk 2 above)
Comment (echarles) [reply experiments/cell/src/index.ts:26]: This is standard api to have fine-grained measures

Comment (goanpeca) [reply experiments/cell/src/index.ts:26]: 👍

File: experiments/cell/style/index.css (hunk 3 above)
Comment (echarles) [reply experiments/cell/style/index.css:14]: This is a rest from previous attemps. As this brings side effects on the scrollbar, we do not retain that. Moving this out.

File: experiments/monaco/README.md (hunk 4 above)
Comment (echarles) [reply experiments/monaco/README.md]: Fixed.

**Review by echarles — COMMENTED** [review]
LGTM

**Pull request merged**